jbpm-designer-distribution-wars/src/main/assembly/assembly-standalone-tomcat-8_0.xml

```diff
@@ -3,7 +3,7 @@
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
 
-  <id>tomcat7.0</id>
+  <id>tomcat8.0</id>
```

Comment:
Good point. Be it my call, I would remove the distribution altogether. I doubt it even deploys anymore.

I'll at least fix the "dot" for now. Although I am not completely sure the comment applies anymore. In any case having just tomcat8 would be aligned with the rest, so will do that.